packages/SwingSet/src/kernel/kernel.js

```diff
@@ -797,7 +842,10 @@ export default function buildKernel(
         kernelKeeper.decrementRefCount(message.kpid, `deq|notify`);
         policyInput = await processNotify(message);
       } else if (message.type === 'create-vat') {
+        // creating a new dynamic vat will immediately do start-vat
         policyInput = await processCreateVat(message);
+      } else if (message.type === 'startVat') {
```

Comment:
I've got a slight preference for kebab-case in these run-queue events (`start-vat` not `startVat`), but it's not worth putting any significant time into changing. The delivery to the vat needs to be camel case, though, because it maps closely to a method name.

Reply:
While I think I agree with your aesthetic instincts on this, we've already got a number of other run queue items that have already bought into camel case (I think, as you point out, to match the corresponding function name), so the only actual kebab-case one is `create-vat`.  If anything I would argue for changing _it_ just for the sake of consistency.  But I think for now this is one of those let sleeping dogs lie things.